src/mpi/init/initthread.c

```diff
@@ -122,8 +140,8 @@ int MPIR_Init_thread(int *argc, char ***argv, int required, int *provided)
     mpi_errno = MPIR_Datatype_init_predefined();
     MPIR_ERR_CHECK(mpi_errno);
 
-    int thread_provided = 0;
-    mpi_errno = MPID_Init(argc, argv, required, &thread_provided);
+    /* **** */
```

Comment:
Was there intention to add some comment here?

Reply:
I just felt like it needs a divider marker.

Reply:
OK, got it. Is there some high-level phase change here we could possibly note? Like we are transitioning from device-independent, local init -> device-specific, non-local init.